openedx/core/djangoapps/course_groups/models.py

```diff
@@ -36,6 +36,11 @@ class Meta:
     GROUP_TYPE_CHOICES = ((COHORT, 'Cohort'),)
     group_type = models.CharField(max_length=20, choices=GROUP_TYPE_CHOICES)
 
+    RANDOM = 'random'
+    MANUAL = 'manual'
+    ASSIGNMENT_TYPE_CHOICES = ((RANDOM, 'Random'), (MANUAL, 'Manual'),)
+    assignment_type = models.CharField(max_length=20, choices=ASSIGNMENT_TYPE_CHOICES, default=MANUAL)
```

Comment:
because we are looking at reusing the CourseUserGroup table/model for other student grouping features, it seems a little awkward to add a column that is very cohort-specific.  I don't really have a great alternative to suggest though.